src/pages/HomePage/Categories.jsx

```diff
@@ -0,0 +1,120 @@
+import "../../styles/Categories.css"
+export const Categories=()=>{
+    return(
+        <div className="shopping-section">
```

Comment:
Categories need to be fetched from backend using api call

Reply:
Made changes to the folder structure, used index js in all folders for export. Categories from backend using API call done... Please re-review.

Reply:
Approved